src/AppBundle/Controller/UserController.php

```diff
@@ -73,4 +74,19 @@ public function avatarAction($username, Request $request)
 
         return $response;
     }
+
+    /**
+     * @Route("/register/resend-email", name="register_resend_email")
```

Comment:
The route should be limited to `POST`
See https://symfony.com/doc/current/routing/requirements.html#adding-http-method-requirements